internal/operators/osc/manifest.go

```diff
@@ -0,0 +1,159 @@
+	openshiftManifests["50_openshift-osc_subscription.yaml"] = oscSubscription
+
+	return openshiftManifests, oscConfigMap, nil
+}
+
+func executeTemplate(data map[string]string, contentName, content string) ([]byte, error) {
+	tmpl, err := template.New(contentName).Parse(content)
+	if err != nil {
+		return nil, err
+	}
+	buf := &bytes.Buffer{}
+	err = tmpl.Execute(buf, data)
+	if err != nil {
+		return nil, err
+	}
+	return buf.Bytes(), nil
+}
+
+func getSubscription(namespace, subscription, source, sourceName string) ([]byte, error) {
+	data := map[string]string{
+		"OPERATOR_NAMESPACE":         namespace,
+		"OPERATOR_SUBSCRIPTION_NAME": subscription,
+		"OPERATOR_SOURCE":            source,
+		"OPERATOR_SOURCE_NAME":       sourceName,
+	}
+	return executeTemplate(data, "oscSubscription", oscSubscription)
+}
+
+func getNamespace(namespace string) ([]byte, error) {
+	data := map[string]string{
+		"OPERATOR_NAMESPACE": namespace,
+	}
+	return executeTemplate(data, "oscNamespace", oscNamespace)
+}
+
+func getOperatorGroup(namespace string) ([]byte, error) {
+	data := map[string]string{
+		"OPERATOR_NAMESPACE": namespace,
+	}
+	return executeTemplate(data, "oscGroup", oscGroup)
+}
+
+func getConfigMap(namespace string, layeredImageDeployment string) ([]byte, error) {
+	data := map[string]string{
+		"OPERATOR_NAMESPACE":                   namespace,
+		"OPERATOR_IS_LAYERED_IMAGE_DEPLOYMENT": layeredImageDeployment,
+	}
+
+	if configMap, err := executeTemplate(data, "oscConfigMap", oscConfigMap); err != nil {
+		return nil, err
+	} else {
+		if layeredImageDeployment == "true" {
+			LayeredImageDeploymentConfigMap, err := getLayeredImageDeployConfigMap(namespace)
+			if err != nil {
+				return nil, err
+			}
+			configMap = append(configMap, []byte("\n---\n")...)
+			configMap = append(configMap, LayeredImageDeploymentConfigMap...)
+
+		}
+		return configMap, nil
+	}
+}
+
+func getLayeredImageDeployConfigMap(namespace string) ([]byte, error) {
+	data := map[string]string{
+		"OPERATOR_NAMESPACE":    namespace,
+		"OPERATOR_KERNEL_ARGS":  KernelArgs,
+		"OPERATOR_OS_IMAGE_URL": LayeredImageUrl,
+	}
+	return executeTemplate(data, "oscLayeredImageDeployConfigMap", oscLayeredImageDeployConfigMap)
+}
+
+const oscNamespace = `
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{.OPERATOR_NAMESPACE}}
+`
+
+const oscGroup = `
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: sandboxed-containers-operator-group
+  namespace: {{ .OPERATOR_NAMESPACE }}
+spec:
+  targetNamespaces:
+    - {{ .OPERATOR_NAMESPACE }}
+`
+
+const oscSubscription = `
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: {{.OPERATOR_SUBSCRIPTION_NAME}}
+  namespace: {{.OPERATOR_NAMESPACE}}
+spec:
+  channel: stable
+  source: {{.OPERATOR_SOURCE}}
+  sourceNamespace: openshift-marketplace
+  name: {{.OPERATOR_SOURCE_NAME}}
+  installPlanApproval: Automatic
+  startingCSV: startingCSV: sandboxed-containers-operator.v1.7.0
```

Comment:
there  is a typo here - but I perfer if we don't have static versions  - this can break it later releases 
so we can remove the `statingCSV`
```suggestion
  startingCSV: sandboxed-containers-operator.v1.7.0
```

Reply:
Removed the startingCSV